core/templates/pages/thanks-page/thanks-page.component.spec.ts

```diff
@@ -16,29 +16,28 @@
  * @fileoverview Unit tests for thanks page component.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// App.ts is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 
-require('pages/thanks-page/thanks-page.component.ts');
+import { ThanksPageComponent } from './thanks-page.component';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service.ts';
 
 describe('Thanks page', function() {
-  var ctrl = null;
+  let component: ThanksPageComponent;
+  let fixture: ComponentFixture<ThanksPageComponent>;
 
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [ThanksPageComponent],
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+    fixture = TestBed.createComponent(ThanksPageComponent);
+    component = fixture.componentInstance;
```

Comment:
Move this into a different beforeEach, Testbed initialization should be separate from as we had in ajs tests. In the old tests, we the component controller was assigned in a different beforeEach. Let's do this for consistency.

Reply:
Done